amf-cli/js/typings/amf-client-js.d.ts

```diff
@@ -1492,6 +1501,8 @@ declare module 'amf-client-js' {
 
     withPayloads(payloads: Array<Payload>): this
 
+    withPayload(): Payload
```

Comment:
why add this ?

Reply:
The typings compile, I'm adding that method because it's part of the `Message` class. Also, every class that extends it already has that method (`Request`, `Response`).